packages/xrpl-crypto/.eslintrc.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
+  root: false,
+
+  parser: '@typescript-eslint/parser', // Make ESLint compatible with TypeScript
+  parserOptions: {
+    // Enable linting rules with type information from our tsconfig
+    tsconfigRootDir: __dirname,
+    project: ['./tsconfig.json', './tsconfig.eslint.json'],
+
+    sourceType: 'module', // Allow the use of imports / ES modules
+
+    ecmaFeatures: {
+      impliedStrict: true, // Enable global strict mode
+    },
+  },
+
+  // Specify global variables that are predefined
+  env: {
+    browser: true, // Enable browser global variables
+    node: true, // Enable node global variables & Node.js scoping
+    es2020: true, // Add all ECMAScript 2020 globals and automatically set the ecmaVersion parser option to ES2020
+    jest: true, // Add Jest testing global variables
+  },
+
+  plugins: [],
+  extends: ['@xrplf/eslint-config/base'],
+
+  rules: {
+    // ** TODO **
+    // all of the below are turned off for now during the migration to a
+    // monorepo. They need to actually be addressed!
+    // **
```

Comment:
This TODO is out of date for this. Might be worth rephrasing here to something like:

```suggestion
    // ** TODO **
    // All of the below are turned off for now. Decide which ones are worth turning on.
    // **
```